src/app/project/list/project-list.component.ts

```diff
@@ -85,4 +85,8 @@ export class ProjectListComponent implements OnInit {
         error => console.log(error)
       );
   }
+
+  goBack(): void {
+    window.history.back();
```

Comment:
try to avoid using this - instead make use of the Angular router to go where the user is supposed to next.